docs/docusaurus.config.js

```diff
@@ -136,6 +136,10 @@ const config = {
             to: '/ci-cd-automation/github-actions-pipeline',
             from: ['/ci-cd-automation/ci-cd-best-practices' /*, '/docs/legacyDocFrom2016'*/],
           },
+          {
+            to: '/live-examples/pokeshop',
+            from: ['/pokeshop', '/pokeshop/' /*, '/docs/legacyDocFrom2016'*/],
```

Comment:
This redirect is not working right now, I'll investigate more later to understand what happened.